org.tryton.Tryton-70.yml

```diff
@@ -0,0 +1,37 @@
+id: org.tryton.Tryton-70
+sdk: org.gnome.Sdk
+runtime: org.gnome.Platform
+runtime-version: '45'
+command: tryton
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=all
+  - --share=network
+  - --filesystem=home
+modules:
+  - python3-tryton.json
+  - name: GoocanvasDependency
+    buildsystem: autotools
+    make-install-args:
+      - pyoverridesdir=/app/lib/python3.10/site-packages/gi/overrides
```

Comment:
did you even test it? the GNOME 45 runtime has python 3.11, so this path is incorrect.

Reply:
Yes I test it.
It is strange because it does compile with it and with 3.11 path, but fails without it.